public_html/script.js

```diff
@@ -39,6 +39,8 @@ var MessageRate = 0;
,[5021696,5025792,"Ireland"],[7569408,7602176,"Israel"],[3145728,3407872,"Italy"],[778240,782336,"Jamaica"],[8650752,8912896,"Japan"],[7602176,7634944,"Jordan"],[6828032,6829056,"Kazakhstan"],[311296,315392,"Kenya"],[13164544,13165568,"Kiribati"],[7364608,7368704,"Kuwait"],[6295552,6296576,"Kyrgyzstan"],[7372800,7376896,"Laos"],[5254144,5255168,"Latvia"],[7634944,7667712,"Lebanon"],[303104,304128,"Lesotho"],[327680,331776,"Liberia"],[98304,131072,"Libya"],[5258240,5259264,"Lithuania"],[5046272,5047296,"Luxembourg"],[344064,348160,"Madagascar"],[360448,364544,"Malawi"],[7667712,7700480,"Malaysia"],[368640,369664,"Maldives"],[376832,380928,"Mali"],[5054464,5055488,"Malta"],[9437184,9438208,"Marshall_Islands"],[385024,386048,"Mauritania"],[393216,394240,"Mauritius"],[851968,884736,"Mexico"],[6819840,6820864,"Micronesia"],[5062656,5063680,"Monaco"],[6823936,6824960,"Mongolia"],[5332992,5334016,"Serbia_and_Montenegro"],[131072,163840,"Morocco"],[24576,28672,"Mozambique"],[7356416,7360512,"Myanmar"],[2101248,2102272,"Namibia"],[13148160,13149184,"Nauru"],[7380992,7385088,"Nepal"],[4718592,4751360,"Netherlands"],[13107200,13139968,"New_Zealand"],[786432,790528,"Nicaragua"],[401408,405504,"Niger"],[409600,413696,"Nigeria"],[4685824,4718592,"Norway"],[7389184,7390208,"Oman"],[7733248,7766016,"Pakistan"],[6832128,6833152,"Palau"],[794624,798720,"Panama"],[9011200,9015296,"Papua_New_Guinea"],[15237120,15241216,"Paraguay"],[15253504,15257600,"Peru"],[7700480,7733248,"Philippines"],[4751360,4784128,"Poland"],[4784128,4816896,"Portugal"],[434176,435200,"Qatar"],[7438336,7471104,"South_Korea"],[5262336,5263360,"Moldova"],[4849664,4882432,"Romania"],[1048576,2097152,"Russian_Federation"],[450560,454656,"Rwanda"],[13156352,13157376,"Saint_Lucia"],[770048,771072,"Saint_Vicent_and_the_Grenadines"],[9445376,9446400,"Samoa"],[5242880,5243904,"San_Marino"],[7405568,7438336,"Saudi_Arabia"],[458752,462848,"Senegal"],[475136,476160,"Seychelles"],[483328,484352,"Sierra_Leone"],[7766016,7798784,"Singapore"],[5266432,5267456,"Slovakia"],[5270528,5271552,"Slovenia"],[491520,495616,"Somalia"],[32768,65536,"South_Africa"],[3407872,3670016,"Spain"],[7798784,7831552,"Sri_Lanka"],[507904,512000,"Sudan"],[819200,823296,"Suriname"],[499712,500736,"Swaziland"],[4882432,4915200,"Sweden"],[4915200,4947968,"Switzerland"],[7831552,7864320,"Syria"],[9015296,9016320,"Taiwan"],[5328896,5329920,"Tajikistan"],[8912896,8945664,"Thailand"],[557056,561152,"Togo"],[13160448,13161472,"Tonga"],[811008,815104,"Trinidad_and_Tobago"],[163840,196608,"Tunisia"],[4947968,4980736,"Turkey"],[6297600,6298624,"Turkmenistan"],[425984,430080,"Uganda"],[5275648,5308416,"Ukraine"],[9003008,9007104,"UAE"],[4194304,4456448,"United_Kingdom"],[10485760,11534336,"United_States_of_America"],[15269888,15273984,"Uruguay"],[5274624,5275648,"Uzbekistan"],[13172736,13173760,"Vanuatu"],[884736,917504,"Venezuela"],[8945664,8978432,"Vietnam"],[8978432,8982528,"Yemen"],[4980736,5013504,"Yugoslavia"],[16384,17408,"Zimbabwe"]];
```

Comment:
Let's move this static data to a separate file.

Also, it'd be a bit more readable if:

 * the constants were in hex, since that's the convention for these addresses elsewhere
 * they were objects not arrays